src/components/trail.style.js

```diff
@@ -1,11 +1,43 @@
 export default {
   container: {
-    flex: 1
+    flex: 1,
+  },
+  mapView: {
+      flex: 1,
+      
+      top: 0,
```

Comment:
Yeah i Just confirmed, You can remove all the top, bottom, left, and right properties on these rules. Jut keep flex and height props.